Header header

float32[3] proximalJointAngle
float32[3] distalJointAngle
float32 fingerSpread
